piptools/utils.py

```diff
@@ -405,9 +414,9 @@ def get_required_pip_specification() -> SpecifierSet:
     Returns pip version specifier requested by current pip-tools installation.
     """
     project_dist = get_distribution("pip-tools")
-    requirement = next(  # pragma: no branch
+    requirement = next(
         (r for r in project_dist.requires() if r.name == "pip"), None
-    )
+    )  # pragma: no branch
```

Comment:
This doesn't seem related to this PR.

Reply:
This appears to be an artifact of pre-commit reorganizing code.

Reply:
That's interesting. I wonder why this didn't happen in the PRs bumping the hooks.. Anyway, I'll mark this thread as resolved, it's not a blocker.